NEWS.md

```diff
@@ -7,6 +7,8 @@
 -   In the when-then-otherwise expressions, the last `$otherwise()` is now optional,
     as in Python Polars. If `$otherwise()` is not specified, rows that don't respect
     the condition set in `$when()` will be filled with `null` (#836).
+-   `<DataFrame>$head()` and `<DataFrame>$tail()` methods now support negative
```

Comment:
also the case for `LazyFrame`, right?

Reply:
> also the case for `LazyFrame`, right?

No, LazyFrame does not know the total number of rows, so negative values are an error.

Reply:
Right